### ISSUE TITLE
Decide Frontend Framework and Initialize Shadcn-based React App

### ISSUE THREAD

**Issue by iamDyeus** [braydenidzenga/Dev-Dialogue#11]
What frontend framework are we moving forward with?
- Next.js
- Vite (TypeScript or JavaScript)
Once decided, I'll assist in setting up a basic Shadcn-based frontend client!

**Comment by codewithjk** [comment]
Next js

**Comment by braydenidzenga** [comment]
@iamDyeus I was planning on using ReactJS to build the front end, with material ui to style the components.

**Comment by Anurag12133** [comment]
@iamDyeus I suggest use Acertinity UI with react it is easy and very much compatible with react
Here is link: https://ui.aceternity.com/

**Comment by MrCyberlord** [comment]
I agree @iamDyeus, @braydenidzenga would love to contribute if React is used.

**Comment by braydenidzenga** [comment]
@iamDyeus Sounds like a good idea. I haven't worked with shadcn before. Would you be able to initialize the frontend in a folder called frontend in the base directory.

**Comment by iamDyeus** [comment]
> @iamDyeus Sounds like a good idea. I haven't worked with shadcn before. Would you be able to initialize the frontend in a folder called frontend in the base directory.

Okay, kindly assign the isssue to me.

**Comment by braydenidzenga** [comment]
@iamDyeus would you be able to add setup instructions to the documentation in your pr.

**Comment by iamDyeus** [comment]
> @iamDyeus would you be able to add setup instructions to the documentation in your pr.

okay i'll do it

**Comment by kolapowariz** [comment]
@iamDyeus do you want me to work with you

**Comment by combfreak45** [comment]
@iamDyeus I would like to work on frontend part with ReactJs and TailwindCSS

**Comment by MrCyberlord** [comment]
@iamDyeus   I would also like to work on it.

**Comment by braydenidzenga** [comment]
@kolapowariz @MrCyberlord @combfreak45 I appreciate you guys for wanting to contribute. After this issue is completed I will be opening a bunch of new ones for the front end so feel free to comment on any of those when they are open and I will assign them to you.

**Comment by iamDyeus** [comment]
@kolapowariz @MrCyberlord @combfreak45 Sure, we can collaborate but likely on issues with more tasks to do. This issue is just about setting up a basic template to kickstart the frontend.

**Comment by MrCyberlord** [comment]
@iamDyeus  yes, exactly!
